docs/launch-with/galaxy-launch-with.rst

```diff
@@ -11,29 +11,35 @@ While browsing Dockstore
 
 Dockstore has a fully featured faceted search to effectively search for workflows. On the left panel, you can choose from several options to narrow down your search. Searching for workflows can be done using a specific search term. Results can be narrowed down based on the language of the workflow, author, organization, source control used among others.
 
-Once you have clicked on a workflow, you will see a Launch with Galaxy button on the right.
+Once you have clicked on a Galaxy-compatible workflow, you will see a Launch with Galaxy button on the right.
```

Comment:
Why "Galaxy-compatible" rather than just "Galaxy"?

Reply:
This was one of Enis's edits but I think we can use just Galaxy. There are other places in the documentation where its just Galaxy and Enis didnt suggest changing it. This must be just a writing quirk.